docs/config/README.md

```diff
@@ -462,6 +462,7 @@ it would have been `nixpkgs/pkgs`.
 | --------------------------- | ------- | ---------------------------------------------------------------------------------------- |
 | `fish_style_pwd_dir_length` | `0`     | The number of characters to use when applying fish shell pwd path logic.                 |
 | `use_logical_path`          | `true`  | Displays the logical path provided by the shell (`PWD`) instead of the path from the OS. |
+| style                       |         | Mirrors the value of option `style`                                                      |
```

Comment:
I think this should be added to the variable section.

Reply:
Not sure I understand. This is the variable section and you refer to `style`?

Reply:
No, this is a table for advanced configurations. You may want to write another table for the details of format-string variables.

Reply:
Oh I see. I created a new section and moved `style` in there (along with `path` as well, part of the variables for the string format).

I also renamed that advanced option column header to `Advanced Option` instead of `Variable`.